backend/services/syson-services/src/main/java/org/eclipse/syson/services/MultiLineLabelSwitch.java

```diff
@@ -82,175 +85,153 @@ public MultiLineLabelSwitch(LabelService labelService) {
 
     @Override
     public String caseElement(Element object) {
-        StringBuilder label = new StringBuilder();
-        label.append(this.labelService.getIdentificationLabel(object));
-        return label.toString();
+        return this.labelService.getIdentificationLabel(object);
     }
 
     @Override
     public String caseAcceptActionUsage(AcceptActionUsage object) {
-        StringBuilder label = new StringBuilder();
-        label
-                .append(this.getBasicNamePrefix(object))
-                .append(LabelConstants.OPEN_QUOTE)
-                .append(this.reference(object))
-                .append("accept action")
-                .append(LabelConstants.CLOSE_QUOTE)
-                .append(LabelConstants.CR)
-                .append(this.caseElement(object))
-                .append(this.multiplicityRange(object))
-                .append(this.labelService.getTypingLabel(object))
-                .append(this.labelService.getRedefinitionLabel(object))
-                .append(this.labelService.getSubsettingLabel(object))
-                .append(this.labelService.getValueLabel(object))
-                .append(LabelConstants.CR)
-                .append(this.acceptActionUsagePayloadDetails(object))
-                .append(this.acceptActionUsageReceiverDetails(object));
-        return label.toString();
+        String label = this.getBasicNamePrefix(object)
```

Comment:
Why are you changing the whole file here?
Please let it as it was.

Reply:
Done